static/main.css

```diff
@@ -234,69 +220,100 @@ textarea:focus {
     opacity: 0;
   }
 }
-footer {
+.Footer {
   background-color: #c49542;
```

Comment:
If this is a common color -- put common colors into stylus variables  (stylus/variables)